.github/workflows/conventional-commit-pr-title-reminder.yml

```diff
@@ -0,0 +1,57 @@
+name: Conventional commit PR Title Reminder
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - enterprise/frontend/src/embedding-sdk/**
```

Comment:
We don't need to use [dorny/paths-filter](https://github.com/dorny/paths-filter) unless we have more complicated path requirements. But this will do for now.